core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/resources.css

```diff
@@ -1,29 +1,28 @@
 #visible-am-container {
-    float: right;
+    height: 100%;
 }
+
 #visible-am-button {
-    text-align: center;
-    font-size: 20px;
-    font-weight: bold;
-    background-color: #e01716;
-    color: #fff;
-    margin: 0;
-    line-height: 40px;
-    text-decoration: none;
-    min-width: 2em;
-    display: inline-block;
     position: relative;
-    transition: all .1s;
-}
-#visible-am-button:hover, #visible-am-button:focus, #visible-am-button:active {
-    background: #e23635;
 }
-#visible-am-container.visible #visible-am-button {
-    box-shadow: inset 0px 1px 14px rgba(0,0,0,.5);
+
+#visible-am-button .am-monitor__count {
+    display: inline-block;
+    display: inline-flex;
```

Comment:
Could you put a comment explaining why you put the two there?

I imagine it's due to the potential lack of support for the inline-flex but in  https://caniuse.com/#feat=flexbox, it seems that all the supported browsers are ok with this.

Reply:
I just do it to be extra defensive. In my experience I learned to be extra paranoid when adding flexbox fallbacks. Once we add postcss most of this can be avoided. 

I can undo these changes if you prefer.

Reply:
> I can undo these changes if you prefer.

Now that we have the explanation, if someone dig into the code they will find this comment and understand. That's good.